e2e/keywords/common.resource

```diff
@@ -48,6 +48,10 @@ Set disk path based on host provider and architecture
 
 Enable v2 data engine and add block disks
     ${V2_DATA_ENGINE_ENABLED}=    get_setting    v2-data-engine
+    ${V1_DATA_ENGINE_ENABLED}=    get_setting    v1-data-engine
```

Comment:
Could you help elaborate on this?

Does this mean: 

If `v1-data-engine` is `false`, set `v1-data-engine` to `false`?

Reply:
Oh, typo here. Fixed

Reply:
Could I know why it's necessary to disable `v1-data-engine` when testing a v2 volume?

Reply:
This means it's impossible to run an integration test that tests both v1 and v2 volumes at the same time.

Also, if I remember correctly, v2 backing images still depend on v1 disks.

Reply:
Removed